app/models/drop_outstanding_reviews.rb

```diff
@@ -0,0 +1,21 @@
+class DropOutstandingReviews  < DeadlineType
+
+	def email_list(assignment_id)
+    emails = drop_outstanding_reviews(assignment_id)
+    emails
+	end
+
+  def drop_outstanding_reviews(assignment_id)
+    emails =[]
+    reviews = ResponseMap.where(reviewed_object_id: assignment_id)
+    for review in reviews
+      review_has_began = Response.where(map_id: review.id)
+        if review_has_began.size.zero?
+          review_to_drop = ResponseMap.where(id: review.id)
```

Comment:
line 14 is a waste, you can skip it and do line 15 as: `review.destroy`
This save one db query.
 But again the _review_ is a bad name..